hc/front/tests/test_add_check.py

```diff
@@ -1,5 +1,8 @@
 from hc.api.models import Check
+
 from hc.test import BaseTestCase
+from hc.api.models import Channel
```

Comment:
It's easier to track imports from the same module if we collect such imports together. We should import Channel together with Check.